app/src/main/res/layout/play_chapter_view.xml

```diff
@@ -37,10 +44,11 @@
       android:id="@+id/chapter_name"
       android:layout_width="0dp"
       android:layout_height="wrap_content"
-      android:layout_marginStart="8dp"
-      android:layout_marginEnd="8dp"
-      android:text="@{chapterSummary.name}"
-      android:textColor="@color/oppiaPrimaryText"
+      android:layout_marginStart="4dp"
+      android:layout_marginEnd="4dp"
```

Comment:
Should be atleast 16dp else please update chapter_container padding end to 16dp

Reply:
set a bigger text and see how UI looks

Reply:
![Screenshot_1573664886](https://user-images.githubusercontent.com/9396084/68787111-f67c6600-0666-11ea-9c2b-0408f147b2d8.png)